libcontainerd/remote_linux.go

```diff
@@ -342,7 +342,7 @@ func (r *remote) runContainerdDaemon() error {
 	// Start a new instance
 	args := []string{"-l", r.rpcAddr, "--runtime", "docker-runc"}
 	if r.debugLog {
-		args = append(args, "--debug")
+		args = append(args, "--debug", "--metrics-interval=0")
```

Comment:
I think we should allow people to enable and disable them independently, otherwise it's a pointless feature.